src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -177,9 +177,9 @@ private async void NewLayoutButton_Click(object sender, RoutedEventArgs e)
             foreach (LayoutModel customModel in MainWindowSettingsModel.CustomModels)
             {
                 string name = customModel.Name;
-                if (name.StartsWith(defaultNamePrefix))
+                if (name.StartsWith(defaultNamePrefix, StringComparison.Ordinal))
```

Comment:
Shouldn't it be `CurrentCulture`? It's localized.

Reply:
Switched it to `CurrentCulture`